plugin/togglecursor.vim

```diff
@@ -38,8 +38,8 @@ let s:supported_terminal = ''
 if !has("gui_running")
     if $TERM_PROGRAM == "iTerm.app" || exists("$ITERM_SESSION_ID")
                 \ || $XTERM_VERSION != ""
-                " \ || $VTE_VERSION != ""
-        " iTerm, xterm, and future VTE based terminals support DESCCUSR.
+                \ || str2nr($VTE_VERSION) >= 3900
```

Comment:
Nice to see support has finally shown up!  What are you running that you have access to this now?